### PR TITLE
Issue 233 style

### PR DESCRIPTION
**Pull request by Ian-Stewart-Binks**
I have made some small style changes to my other pull request #248. 
These changes include:
    * Converting messy quotation usage into more readable format (things like `"...\""` -> `'..."'`.
    * Replacing some loops in `svg_parser.hs` with `map` calls.

### CODE REVIEW

File: utilities/svg_parsing/svg_parser.py [hunk 1]
```diff
@@ -42,62 +42,70 @@ def read_svg():
                     rects.remove(rect_2)
                     final_rects.append(rect_1)
 
+
 def find_all_and_process(soup, tag, fn):
-    for elem in soup.find_all(tag):
-        fn(elem)
+    list(map(fn, soup.find_all(tag)))
```
Comment (david-yz-liu) [inline utilities/svg_parsing/svg_parser.py]: This is actually less Pythonic.

**Comment by Ian-Stewart-Binks** [comment]
Adding in triple quotes for the header is possible, but in order to leave out (whitespace)[http://stackoverflow.com/questions/1520548/how-does-pythons-triple-quote-string-work] it looks hideous.

**Comment by david-yz-liu** [comment]
Okay, makes sense.

**Comment by david-yz-liu** [comment]
@Ian-Stewart-Binks Looks like there's a merge conflict; could you please fix this?

**Comment by david-yz-liu** [comment]
@Ian-Stewart-Binks Looks like something in the javascript doesn't like the hybrid ID's with slashes in them - e.g., hover over CSC404, and nothing else will be highlighted.

**Comment by Ian-Stewart-Binks** [comment]
Can now be merged.

**Pull request merged**